src/relay/backend/graph_executor_codegen.cc

```diff
@@ -403,64 +405,75 @@ class GraphExecutorCodegen : public backend::MemoizedExprTranslator<std::vector<
     return lhs_storage_id == rhs_storage_id;
   }
 
-  std::vector<GraphNodeRef> GraphAddCallNode(const CallNode* op, const std::string& func_name,
-                                             GraphAttrs attrs) {
+  std::vector<GraphNodeRef> GraphAddCallNode(const CallNode* call_node, GraphAttrs attrs) {
+    Call call = GetRef<Call>(call_node);
+    ICHECK(call->op == CallLoweredOp())
+        << "Non-primitive-call nodes should have been transformed away.\n"
+        << "The graph executor code generator expects all calls to be call_lowered, "
+        << "but found: " << std::endl
+        << PrettyPrint(call);
+
+    // Extract function and arguments from the call_lowered op
+    std::pair<GlobalVar, Array<Expr>> func_and_args = ExtractFunctionAndArgs(call_node);
+    GlobalVar func = func_and_args.first;
+    Array<Expr> call_args = func_and_args.second;
+
+    // std::string func_name = name;
+    std::string func_name = func->name_hint;
+
     std::vector<GraphNodeRef> inputs;
-    for (auto arg : op->args) {
-      auto res = VisitExpr(arg);
-      for (auto nr : res) {
-        inputs.push_back(nr);
+    // Visit all the arguments to call_lowered
+    for (Expr arg : call_args) {
```

Comment:
nit: good practice to const Expr& or const auto& to avoid the copy. not a big deal here but will bite you one day i promise!